lib/spack/llnl/util/_windows/symlink.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```

Comment:
```suggestion
# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

Reply:
Great catch, I wonder where I copied those from and more importantly why CI was passing :thinking: